bool part_is_valid
uint32 part_id
geometry_msgs/Point point
float32 confidence
